backend_plugin/src/main/evaluators/Evaluator.java

```diff
@@ -8,6 +8,7 @@
 import org.eclipse.jface.text.source.IAnnotationModel;
 import org.eclipse.ui.texteditor.ITextEditor;
 import main.listeners.DocumentChangesListener;
+import main.interfaces.FeatureSuggestion;
```

Comment:
Looks like this import isn't used anymore in this class. No biggie right now, but something to think about when we're cleaning/refining stuff

Reply:
If only I had a tool that caught unused imports and suggested how to get rid of them....